torch/profiler/_pattern_matcher.py

```diff
@@ -609,8 +628,8 @@ def report_all_anti_patterns(prof,
     report_dict: Dict = {}
     anti_patterns = [
         ExtraCUDACopyPattern(prof, should_benchmark),
-        ForLoopIndexingPattern(prof, should_benchmark),
-        FP32MatMulPattern(prof, should_benchmark),
+        # ForLoopIndexingPattern(prof, should_benchmark),
```

Comment:
These are still tested, right?

Reply:
Yes, the test entry is not report_anti_patterns, but rather the pattern classes :)